src/raw/cgroup.go

```diff
@@ -16,22 +16,87 @@ import (
 	"github.com/newrelic/infra-integrations-sdk/log"
 )
 
-const (
-	localCgroupPath = "/sys/fs/cgroup" // todo: make it configurable
-)
+const cgroupDir = "/cgroup"
+
+var localCgroupPaths = []string{
+	"/sys/fs/cgroup",
+	"/cgroup",
+}
 
 // cgroupsFetcher fetches the metrics that can be found in cgroups file system
 type cgroupsFetcher struct {
 	cgroupPath string
 	subsystems cgroups.Hierarchy
 }
 
-func newCGroupsFetcher(hostRoot string) *cgroupsFetcher {
-	path := containerToHost(hostRoot, localCgroupPath)
+func newCGroupsFetcher(hostRoot, cgroup string) *cgroupsFetcher {
+	if cgroup != "" {
+		// Prepend cgroup to have higher priority than the predefined paths.
+		localCgroupPaths = append([]string{cgroup}, localCgroupPaths...)
+	}
+
+	cgroupPath, err := detectCgroupPath(localCgroupPaths)
```

Comment:
it's usually a good practice to extract env-var from runtime workflow and pass their values as usual args to constructors or runtime functions

we could improve this on next iteration